pkg/kubelet/dockertools/manager.go

```diff
@@ -384,11 +384,14 @@ func (dm *DockerManager) inspectContainer(id string, podName, podNamespace strin
 
 		terminationMessagePath := containerInfo.TerminationMessagePath
 		if terminationMessagePath != "" {
-			if path, found := iResult.Volumes[terminationMessagePath]; found {
-				if data, err := ioutil.ReadFile(path); err != nil {
-					message = fmt.Sprintf("Error on reading termination-log %s: %v", path, err)
-				} else {
-					message = string(data)
+			for _, mount := range iResult.Mounts {
```

Comment:
I found it was broken before. The test proves that. :)
We don't have test for termination message before, in fact it was broken from 1.1, I just occasionally found it recently. With node e2e test, this tragedy would never happen again! :)